packages/mdc-menu/mdc-menu.scss

```diff
@@ -14,135 +14,58 @@
 // limitations under the License.
 //
 
-@import "@material/animation/variables";
-@import "@material/elevation/mixins";
-@import "@material/theme/mixins";
-@import "@material/theme/variables";
-@import "@material/typography/mixins";
+@import "@material/rtl/mixins";
+@import "@material/list/mixins";
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+@import "./variables";
+@import "./mixins";
+
+// Customize the menu-surface and contained list to match the mdc-menu styles.
+.mdc-menu-surface {
+  @include mdc-list-item-meta-ink-color(text-primary-on-background);
+  @include mdc-list-item-graphic-ink-color(text-primary-on-background);
 
-$mdc-menu-fade-in-duration: .03s;
-$mdc-menu-fade-out-duration: .075s;
-$mdc-menu-scale-duration: .12s;
-
-// postcss-bem-linter: define menu
-.mdc-menu {
-  @include mdc-elevation(2);
-  @include mdc-theme-prop(background-color, background);
-
-  display: none;
-  position: absolute;
-  box-sizing: border-box;
-  min-width: 170px;
-  max-width: calc(100vw - 32px);
-  max-height: calc(100vh - 32px);
-  margin: 0;
-  padding: 0;
-  transform: scale(1);
-  transform-origin: top left;
-  border-radius: 2px;
-  opacity: 0;
-  white-space: nowrap;
-  overflow-x: hidden;
-  overflow-y: auto;
-  will-change: transform, opacity;
-  z-index: 4;
-
-  &:focus {
-    outline: none;
-  }
-
-  &--animating-open {
-    display: inline-block;
-    transform: scale(.8);
-    transition:
-      opacity $mdc-menu-fade-in-duration linear,
-      transform $mdc-menu-scale-duration $mdc-animation-deceleration-curve-timing-function;
-    opacity: 0;
-    overflow-y: hidden;
-  }
-
-  &--open {
-    display: inline-block;
-    transform: scale(1);
-    opacity: 1;
-  }
-
-  &--animating-closed {
-    display: inline-block;
-    transition: opacity $mdc-menu-fade-out-duration linear;
-    opacity: 0;
-    overflow-y: hidden;
-  }
-
-  &__items {
-    box-sizing: border-box;
-    transform: scale(1);
-    overflow-x: hidden;
-    overflow-y: auto;
-    will-change: transform;
-
-    // stylelint-disable plugin/selector-bem-pattern
-    > .mdc-list-item {
-      cursor: pointer;
-    }
-    // stylelint-enable plugin/selector-bem-pattern
-
-    .mdc-menu--animating & {
-      overflow-y: hidden;
-    }
+  .mdc-list {
```

Comment:
Doesn't this violate BEM by creating increasing specificity with descendent selectors?